corehq/form_processor/models.py

```diff
@@ -258,9 +259,7 @@ class AbstractAttachment(models.Model):
 
     @property
     def filepath(self):
-        if getattr(settings, 'IS_TRAVIS', False):
-            return os.path.join('/home/travis/', str(self.attachment_id))
-        return os.path.join('/tmp/', str(self.attachment_id))
+        return os.path.join(gettempdir(), str(self.attachment_id))
```

Comment:
We might be able to set an environment variable such as `TMPDIR`. I noticed in looking at the history of this file that you had tried `/hom/travis/tmp`. Do you remember why that didn't work?

Interestingly the documentation for `gettempdir()` (well actually `tempdir`) says this:
> If tempdir is unset or None at any call to any of the above functions, Python searches a standard list of directories and sets tempdir to the first one which the calling user can create files in.

Do you know, when you ran into the read-only `/tmp` was it on the old non-container-based travis or the new setup?